frontend/packages/ux-editor/src/components/leftMenu/LayoutSetsContainer.module.css

```
@@ -0,0 +1,3 @@
+.dropDownContainer {
+  margin: 10px;
```

Vi bør synkronisere margen med de andre tingene i kolonnen. Foreslår at vi bruker `---fds-spacing-5` inntil videre, siden det er det samme som brukes i `Accordion`-ene under. På sikt bør vi kanskje ha en intern variabel for dette som vi bruker på alt, i tilfelle standardoppsettet i Accordion skulle endre seg (dessuten er nivå 5 litt stort for en så smal kolonne).
`margin-bottom`-kan du sette til `0` siden det allerede er en toppmarg på knappen under.
```suggestion
  margin: var(--fds-spacing-5);
  margin-bottom: 0;
```
Det mest ryddige her ville egentlig vært å fjerne alle marger både her og på knappen, og heller sette margen på nivået over, og bruke `gap` til å få avstand mellom elementene. Da ville alt som har med avstand å gjøre vært på samme sted, og vi ville unngått problemer med marger som settes dobbelt o.l.